contributors/triage/guideline.md

```diff
@@ -0,0 +1,62 @@
+# Kubernetes Issue Triage Guidelines
+
+## Purpose
+
+Speed up issue management.
+
+The Kubernetes issues are listed at https://github.com/kubernetes/kubernetes/issues and are identified with labels. For example, an issue that belongs to SIG Network group will eventually be set to label `sig/network`. New issues will start out without any labels. The detailed list of labels can be found at https://github.com/kubernetes/kubernetes/labels. While working on triaging issues you may not have privilege to assign specific label (e.g. `triaged`) and in that case simply add a comment in the issue with your findings.
+
+## Scope
+
+These guidelines serves as a primary document for triaging incoming issues to Kubernetes. SIGs and projects are encouraged to either use these guidelines, or use this as a starting point if necessary. For example if your SIG has specific triaging needs, extend these guidelines. Also, these guidelines, in general, can be applied to manage any repository issues related to the kubernetes development.
+
+## Using the bot
+
+Most people do not have the ability to interact with the github issues and labels, for that we use a bot to manage labelling and triaging. The bot has a set of [commands and permissions](https://github.com/kubernetes/test-infra/blob/master/commands.md) and this document will cover the basic ones.
+
+## Determine if it’s a support request
+
+Sometimes users ask for support requests in issues; these are usually support requests for people who need help configuring some aspect of Kubernetes. These should be directed to our [support structures](https://github.com/kubernetes/community/blob/master/contributors/devel/on-call-user-support.md) and then closed. Also, if the issue is clearly abandoned or in the wrong place, it should be closed. Keep in mind that only issue reporter, assignees and component organization members can `/close` issue. If you do not have such privilege, just comment your findings.
+
+## Find the right SIG(s)
+Components are divided among [Special Interest Groups (SIGs)](https://github.com/kubernetes/community/blob/master/sig-list.md). Find a proper SIG for the ownership of the issue using the bot:
+
+* Typing `/sig network` in a comment should add the sig/network label, for example.
+* Multiword SIGs use dashes, for example `/sig cluster-lifecycle`.
+
+## Validate if the issue is real
+
+Validate if the problem is real by reproducing it. If reproducible, move to the next step of defining priority. You may need to contact the issue reporter in the following cases:
+* Do a quick duplicate search to see if the issue has been reported already. If a duplicate is found, let the issue reporter know it by marking it duplicate.
```

Comment:
Yes, via a comment with a reference to the original issue and request to close it. I will mention it explicitly here for clarity.